.gitignore

```diff
@@ -1,42 +0,0 @@
-.bundle
-.rbenv-version
-.swp
-*.swo
-*.swp
-.sass-cache
-db/*.sqlite3
-db/*.csv
-log/*.log
-log/*.log.lck
-log/*.log.*
-tmp/
-.idea/*
-\#*
-.#*
-*~
-*.~lock.*
-tags
-.emacs.desktop
-.DS_Store
-*.sublime-project*
-spec/javascripts/generated/*
-db/development_structure.sql
-db/test_structure.sql
-db/staging_structure.sql
-db/production_structure.sql
-db/backup
-app/public
-public/system
-public/stylesheets
-public/images
-public/spree
-config/abr.yml
-config/heroku_env.rb
-config/newrelic.yml
-config/initializers/feature_toggle.rb
-config/initializers/db2fog.rb
-NERD_tree*
-coverage
-libpeerconnection.log
-/config/application.yml
-node_modules
```

Comment:
😱

Reply:
don't think we want all this taken out of gitignore, @Duende13